src/MobileDevice/widget/MobileDevice.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="MobileDevice.widget.MobileDevice" offlineCapable="true" needsEntityContext="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Mobile Device</name>
+    <description>Get device information</description>
+    <icon>iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAACLlBMVEUAAAAJCQlqamoICAhiYmIzMzM1NTX///8DAwM0NDRRUVEnJycwMDAAAABISEhZWVkpKSlERER7e3uXl5ehoaGYmJhgYGAdHR1CQkJ5eXmYmJjR0dGamppgYGAsLCwdHR2EhIS3t7dcXFwiIiKioqKhoaFra2stLS0hISGenp7ExMSmpqZcXFwLCwtnZ2e6urpycnJUVFSRkZFsbGx+fn5ycnI8PDx9fX2NjY3CwsJxcXGGhoavr69+fn5GRkZycnKDg4NfX19/f3+2traCgoJPT08VFRUICAgKCgpXV1eNjY2ysrKjo6NdXV1paWlSUlITExMsLCxaWlp+fn6mpqagoKAxMTEAAAAAAAAAAAAAAAAAAAAAAAAAAAAPDw9PT0+NjY2EhIQmJiYAAAAAAAAAAAAAAAAAAAAAAAAMDAwICAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACWlpaUlJTGxsbx8fHKysqmpqa/v7/u7u76+vr4+Pjr6+vZ2dnV1dXq6ur39/fw8PDl5eXk5OTh4eDc3NzNzc3Q0NDg4ODp6Ojt7OzY4eSYpamtraylpaXY2Njc5eiYnqFyoa+JpKypqKjJycmgoKC9vbyUnJ54r7+Nm6C9w8XS1NS3trbAwMDR0dHU1NTS0tKMjIxZWVmDg4Ocn6DAzNDf4ODe3t7IyMjT09PPz8/Ozs6oqKiQkJDLy8vl5eTFxcW9vb28vLy7u7vCwsLW1tau+PASAAAAdXRSTlMAAAAAAAAAAAAAAAAAAi9WFSiG3/fBXBEnh+D5wl0PE8T7fBna4XceGNn91m0TofyoXLi9+88+w/n+s8T1s1aFtVjD9bROCwIIZ9L9+8WqUAwbddn96F8hIhwQAwEdSY7ZzVgyNDMTKC5EQDApEgUNGBsZFATdcWVWAAAAAWJLR0QHFmGI6wAAAAlwSFlzAAAASAAAAEgARslrPgAAAP5JREFUGNNjYAACRl4+fgEmBhhg5hUUEhYRFRNnAXNZGSQkpUrLyiukZWTl2FiBAvIKlVXVNbV1dfWKSuwcDAycyioNjU3NzS2tbe2qaupAFRqaHY2dXd09vX39E7S0dRi4dPUmVnaVT5o8Zeq06TP0DRgMjYxNSmfOmj1n7rz5CxaamjGYL1q8ZOmy5StWrlq9xsJyhhWD9dp169ds2Lhp82IbWzt7B0cGp851W7Yu3rbd2cXVjdvdg4HB08t74Q4fXz//AB6oywODgkN2hoaFR0RGRYMFYmLj4hMSk5JTUlJjY8AiaekZmVnJqcnZOblQTXn5aQWFRcX5JSAOAKOuTkLqtuOUAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDE3LTA2LTE1VDAxOjMzOjA1LTA1OjAwY39ExgAAACV0RVh0ZGF0ZTptb2RpZnkAMjAxNy0wNi0xNVQwMTozMzowNS0wNTowMBIi/HoAAAAZdEVYdFNvZnR3YXJlAEFkb2JlIEltYWdlUmVhZHlxyWU8AAAAAElFTkSuQmCC</icon>
+    <properties>
+        <property key="deviceEntity" type="entity" required="false">
```

Comment:
deviceEntity is not needed, when needsEntityContext is is used the mf and attributes are using the context as param